modules/account/README.md

```diff
@@ -6,9 +6,8 @@ account.
 
 > [!NOTE]
 >
-> Part of a
-> [cold start](https://docs.cloudposse.com/reference-architecture/how-to-guides/implementation/enterprise/implement-aws-cold-start)
-> so it has to be initially run with `SuperAdmin` role.
+> Part of a [cold start](https://docs.cloudposse.com/layers/accounts/prepare-aws-organization/) so it has to be
```

Comment:
Unless we're in a `<ReactPlayer>`, we should not prefix links with `docs.cloudposse.com` because then they don't work in local dev.

```suggestion
> Part of a [cold start](/layers/accounts/prepare-aws-organization/) so it has to be
```

Reply:
but that wont work from the GitHub readme. It's easier to remove `https://docs.cloudposse.com` in docs-collator

Reply:
Oh, I see. Because this is in terraform-aws-components, we're linking to the docs site. Nevermind.